node_lambnik/env-template

```diff
@@ -8,3 +8,9 @@ TILEGARDEN_DB=
 TILEGARDEN_DB_HOST=
 TILEGARDEN_DB_PASSWORD=
 TILEGARDEN_DB_USER=
+
+# Function config information
+DB_TABLE=
+GEOMETRY_FIELD=geom
+EPSG=4326
+TIMEOUT=5
```

Comment:
Probably makes sense to namespace these.  E.g. `TILEGARDEN_DB_TABLE`, `TILEGARDEN_GEOMETRY_FIELD`, `TILEGARDEN_GEOMETRY_EPSG`.  And a more descriptive name for what timeout that's setting.

Reply:
I think I'm going to address this in the CartoCSS PR where I'm already refactoring a lot of the configuration setup.